find_max.rb

```diff
@@ -0,0 +1,19 @@
+
+# Find the maximum element between index 0 and max_index, exclusive
+
+def find_max(array, max_index)
```

Comment:
is max_index the max index of array? if so, could we just find that inside the method instead of passing it in as a parameter?